src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Avoid repeating BASE_URL in several files. Create a file that contains the BASE_URL only once and exports to the files that will use it. Or add it to .env file and import from there. The point is to avoid repeating the constant all over the code, making it harder for code maintenance if eventually the URL changes.